install/install_openadapt.sh

```diff
@@ -14,8 +14,29 @@ RunAndCheck() {
         exit 1
     fi
 }
+
+# Install the required dependencies for the SVG Mixin
+# Specifically for Vtracer and Cairo
+InstallSVGDependencies() {
```

Comment:
@dianzrong can you please clarify how I can test this out? Running the install script isn't quite right because I already have a cloned repo. Should I just manually run the `curl` and `cargo` commands, or is there a better way?

Reply:
Personally I like to run the install script inside the install directory so it creates an OpenAdapt inside the repository. This way it's clear which is the one you've been working in and which is from the install script. This is important because running `cargo install vtracer` for example, is different from running ` RunAndCheck "cargo install vtracer" "install vtracer"`. 

**That being said,** I'm turning this PR into a draft until the mac install script PR gets merged so the brewInstall function can be used to check for and install rust more cleanly.

Reply:
Thanks @dianzrong ! Can you please clarify:

>  This is important because running cargo install vtracer for example, is different from running  RunAndCheck "cargo install vtracer" "install vtracer".

How is it different?

Reply:
RunAndCheck runs the first argument and based on that argument's **output**, prints and/or exits early.  (I have changed it in the mac install script so it only exits early if the first argument's **return value** is not 0.)